pipeline.yml

```diff
@@ -833,6 +877,15 @@ resources:
       branch: main
       commit_verification_keys: ((cloud-gov-pgp-keys))
 
+  - name: deploy-s3-broker-dev-tasks
```

Comment:
We need to create the resources for staging and prod, right?

Reply:
We still need resources for `deploy-s3-broker-staging-tasks` and `deploy-s3-broker-production-tasks`

Reply:
They're there